### PR TITLE
docs: add stackup example for erc20 integration

### PR DESCRIPTION
**Pull request by avarobinson**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
The focus of this PR is to introduce support for ERC20 gas sponsorship using a third-party paymaster like Stackup. 

### Detailed summary
- Added information about using Stackup for ERC20 gas sponsorship in `third-party/paymasters.md`
- Updated FAQ to mention the ability to use third-party paymasters for ERC20 gas sponsorship

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/pages/third-party/paymasters.md [hunk 1]
```diff
@@ -33,3 +33,69 @@ const client = createSmartAccountClient({
...
+  account: {
+    signer,
+  },
+  transport: http("ALCHEMY_RPC_URL"),
+  gasEstimator: async (userOp) => ({
+    ...userOp,
+    callGasLimit: "0x0",
+    preVerificationGas: "0x0",
+    verificationGasLimit: "0x0",
+  }), // Stackup gas estimation
```
Comment (rthomare) [inline site/pages/third-party/paymasters.md:75]: What does stackup gas estiamtion mean? Does the end user need to do anything or are these just dummy values?

**Review by rthomare — DISMISSED** [review]

**Comment by rthomare** [comment]
Looks good but would be helpful to add more clarity for one commend and one todo!

**Review by rthomare — APPROVED** [review]

File: site/pages/resources/faqs.md [hunk 2]
```diff
@@ -6,8 +6,6 @@ description: Learn how to get started with Alchemy's Account Kit, a vertically
 
 # Frequently asked questions
 
-Check out our [SDK discussions](https://github.com/alchemyplatform/aa-sdk/discussions) for more FAQs. Feel free to open a new discussion for any other questions, issues, or feature requests and we will be happy to help :)
```
Comment (moldy530) [inline site/pages/resources/faqs.md:9]: should we not keep this here?

File: site/pages/resources/faqs.md [hunk 3]
```diff
@@ -143,13 +143,6 @@ Gas Manager policies can only be tied to one app. Make sure you are using the AP
 ### Precheck failed: `{ code: -3200, message: 'precheck failed: ...' }`
 
 :::details[Answer]
-If you are using Alchemy's infra for AA:
```
Comment (moldy530) [inline site/pages/resources/faqs.md:146]: shouldn't we keep this?

**Review by moldy530 — COMMENTED** [review]
More generally, I think we should migrate these FAQs to github discussions and start killing this page / eventually have the FAQs link just redirect to the github discussions

File: site/pages/third-party/paymasters.md [hunk 4]
```diff
@@ -33,3 +33,69 @@ const client = createSmartAccountClient({
...
 });
 ```
+
+## ERC20 Gas Sponsorship
+
+We are working on building support for an ERC20 paymaster!
+
+In the meantime, you could use a third-party paymaster, such as [Stackup](https://docs.stackup.sh/docs/paymaster-api), to sponsor ERC-20 gas. Here's an example using Stackup with the Alchemy SDK:
+
+```ts
```
Comment (moldy530) [inline site/pages/third-party/paymasters.md:43]: to make sure this compiles, can you change this to:
```suggestion
```ts twoslash
```

then load the page and make sure that there's no errors in the code preview?

File: site/pages/third-party/paymasters.md [hunk 5]
```diff
@@ -33,3 +33,69 @@ const client = createSmartAccountClient({
...
+import { createClient, http } from "viem";
+import { sepolia } from "viem/chains";
+
+const chain = sepolia;
+// [!code focus:99]
+const stackupClient = createClient({
+  transport: http("https://api.stackup.sh/v1/paymaster/STACKUP_API_KEY"), // TODO: Replace with your stackup API key here (https://docs.stackup.sh/docs/paymaster-api)
+});
+
+const alchemyRpcClient = createAlchemyPublicRpcClient({
```
[the remark this answers is not in the document]
Comment (moldy530) [reply site/pages/third-party/paymasters.md:59]: where is this client used?

**Comment by moldy530** [comment]
closing out as it was done in another branch

**Pull request closed without merging**